h2o-core/src/main/java/water/Job.java

```diff
@@ -301,7 +304,6 @@ public final void update( final long newworked, final String msg) {
     @Override public boolean onExceptionalCompletion(Throwable ex, CountedCompleter caller) {
       if(Job.isCancelledException(ex)) {
         new Barrier1OnCom().apply(Job.this);
-        _barrier = null;
```

Comment:
What is the purpose of this change?

Reply:
this line was just pointless (_barrier is always nullified at the end of the method) -> removed

Reply:
I think it might be a way of making sure the barrier is not accessed in this method (if it was modified). It is useless for the correct code execution, I agree with that, of course.

Reply:
closed